Formula/borgbackup.rb

```diff
@@ -25,14 +24,14 @@ class Borgbackup < Formula
   depends_on "xxhash"
   depends_on "zstd"
```

Comment:
Do we have `pkgconfig` (== the pypi package with that name) installed before setup.py of borgbackup is running?

Reply:
well, if it finds all the stuff it needs even without pkgconfig, you can leave it away.